### ISSUE TITLE
Spam filters

### ISSUE THREAD

**Issue by Mehael** [mkalam-alami/ludumdare-feedback-friends#43]
Some bad guys write just one or two words A LOT to recive 2 or 3 points.
Like "Fantastic! Fantastic! Fantastic!". 
:(

**Comment by ttencate** [comment]
Naturally ;) Whenever you make rules, people will try to game them. (Is that Somebody's Law yet? If not, I claim it!)

My first thought was: we could score each comment based on its gzipped size using `gzcompress`. Such repetitive comments will compress really well. However, then people will just mash their keyboard or even use an actual PRNG. In the end, it's just pathetic behaviour, which like most social problems is difficult or impossible to solve with technical means.

If you could post links to the offending users, we can blacklist them.

**Comment by balohmatevz** [comment]
Because comments they've made show up on their page as well, it's fairly obvious who's done this sort of stuff, and thus easy to call out when reviewing their game.

**Comment by Mehael** [comment]
Hmm.. gzip idea looks surprisingly. I started to think line algorithm, but it's can works. 
I looked all posts of [http://feedback.ld.intricati.com/?event=ludum-dare-36&uid=54139](this man), but found only one semi-illegal comment (for me). So I don't think what we should blacklist he.

I agreed, it's wrong way to fix a problem with algorithms. **Maybe, it can be solved by community-self-moderating system, as part of likes-dislikes system**. If this project will be used for LD in the future. Anyway, thank you for comfortable site.
